test(isolated-declarations): fix multiline comment mismatch in windows

Dunqing opened the pull request:
Fixes https://github.com/oxc-project/oxc/actions/runs/15327109307/job/43124348533

I don't know why it failed in Windows, I just follow the Boshen way to solve the problem.

Dunqing commented:
* **#11373** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/11373?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/11373?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>0-merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/oxc-project?ref=merge-queue-instructions-in-stack-comment&prId=11373).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

Boshen commented:
### Merge activity

* **May 29, 3:24 PM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

Dunqing committed:
test(isolated-declarations): fix multiline comment mismatch in windows (#11373)

Fixes https://github.com/oxc-project/oxc/actions/runs/15327109307/job/43124348533

I don't know why it failed in Windows, I just follow the Boshen way to solve the problem.

Sysix commented:
> I don't know why it failed in Windows, I just follow the Boshen way to solve the problem.

@Dunqing In `oxlint` we have one test excluded in windows because of the span offset for line breaks `\r\n` vs `\n`
https://github.com/oxc-project/oxc/blob/1cd8b9c7e9eab2201ad1fc5ee890f54d6b3be3d7/apps/oxlint/src/output_formatter/mod.rs#L142-L151

I guess this is something similar

Dunqing commented:
> @Dunqing In `oxlint` we have one test excluded in windows because of the span offset for line breaks `\r\n` vs `\n`
> 
> https://github.com/oxc-project/oxc/blob/1cd8b9c7e9eab2201ad1fc5ee890f54d6b3be3d7/apps/oxlint/src/output_formatter/mod.rs#L142-L151
> 
> I guess this is something similar

Thanks for pointing it out! I will take a look.
